pyClient/zeth/encryption.py

```diff
@@ -4,21 +4,36 @@
 #
 # SPDX-License-Identifier: LGPL-3.0+
 
-from nacl.public import PrivateKey, PublicKey  # type: ignore
-import nacl.encoding  # type: ignore
 from typing import NewType
 
-# Thin wrapper around the nacl PublicKey and PrivateKey.  Note nacl doesn't
-# include much type information, so we try to enforce strict types.
+from cryptography.hazmat.primitives.asymmetric.x25519 \
```

Comment:
Just a small thing, but we try to use the convention of ordering imports: local (zeth), external libraries, standard lib (except __future__ which must be first). Would you mind reordering here to be consistent?

Reply:
I have switched them. Sorry, what do you mean with "future"?

Reply:
@riemann89 https://github.com/clearmatics/zeth/blob/develop/pyClient/zeth/contracts.py#L7